controllers/users.js

```diff
@@ -568,6 +568,28 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+const updateRoles = async (req, res) => {
+  try {
+    const result = await userQuery.fetchUser({ userId: req.params.id });
+    if (result?.userExists) {
+      const dataToUpdate = req.body;
+      const userQueryResponse = await userQuery.updateRoles(result.user, dataToUpdate);
+      if (userQueryResponse.isRoleUpdated) {
+        return res.json({
+          message: "role updated successfully!",
+        });
+      } else {
+        return res.boom.conflict("role already exist!");
+      }
```

Comment:
how are we sure that userQueryResponse will return isRoleUpdated false **only** if the user has the same role?

Reply:
from Model file either true return or false
 https://github.com/Real-Dev-Squad/website-backend/pull/1147/files#diff-9a5062203df8184c1d64d655f2dcd16eebeea484b834b0f461771b463037ceefR583

Reply:
Done with discussion